rosa_getting_started/rosa-deleting-cluster.adoc

```diff
@@ -7,4 +7,13 @@ toc::[]
 
 Delete a {product-title} (ROSA) cluster using the `rosa` command-line.
 
+[id="prerequisites_rosa-deleting-cluster"]
+== Prerequisites
+
+* If {product-title} created a VPC, you must remove the following before you delete your cluster:
```

Comment:
```suggestion
* If {product-title} created a VPC, you must remove the following [$PLURAL_NOUN] before you delete your cluster:
```

Better for translation, but nothing in particular is coming to mind. :(

Reply:
@maxwelldb This work?

* If {product-title} created a VPC, you must remove the following items from your cluster before you can successfully delete your cluster: